docs/section6/Rule6-12.md

```diff
@@ -0,0 +1,36 @@
+
+# Lighting - Rule 6-12
+
+**Rule ID:** 6-12  
+**Rule Description:** Proposed building is modeled with daylighting controls  
+**Appendix G Section:** Section 6 Lighting  
+**Appendix G Section Reference:** Section G3.1-6(h) Lighting: Modeling Requirements for the Proposed design  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+## Rule Logic: 
+
+- Check if each space has window or skylight in the building segment in the Proposed model: `For zone_p in P_RMR...zones:`
+
+  - Get surfaces in zone: `surface_p in zone_p.surfaces`
+
+    - Check if the surface is exterior and has openings: `if ( surface_p.adjacent_to == AMBIENT ) AND ( COUNT( surface_p.fenestration_subsurfaces ) >= 1 ): daylight_flag_p == TRUE`
+
```

Comment:
1. Please replace "....==AMBIENT" with "...==EXTERIOR"
2. Suggest to additionally check that subsurface.classificaiton_type!=Door, because majority of doors are opaque, so that dyalight_flag_p== TRUE if zone has subsurfaces other than doors.

Reply:
Updated.